src/dataSources/JSON.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var Null = require('./Null');
+
+/**
+ * @constructor
+ */
+var JSON = Null.extend('JSON',  {
+
+    initialize: function(data, fields, calculators) {
+        this.setData(data, fields, calculators);
+    },
+
+    isNullObject: false,
+
+    getDataIndex: function(y) {
+        return y;
+    },
+
+    /**
+     * @memberOf DataSource.prototype
+     * @param y
+     * @returns {object[]}
+     */
+    getRow: function(y) {
+        return this.data[y];
+    },
+
+    findRow: function(columnName, value) {
```

Comment:
This version of `findRow` is out of date. Please paste in the version from hyper-analytics/js/DataSource.js.